Source/Plugins/EditorModules/SceneView/Modules/SceneView.cs

```diff
@@ -1477,16 +1477,31 @@ private void lockedToolStripMenuItem_Click(object sender, EventArgs e)
 
 		private void gameObjectToolStripMenuItem_Click(object sender, EventArgs e)
 		{
-			// Create the GameObject
-			GameObject obj = this.CreateGameObject(this.objectView.SelectedNode);
-			GameObjectNode objNode = this.FindNode(obj);
+			// Create new objects
+			List<GameObjectNode> newObjects = new List<GameObjectNode>();
+			if (this.objectView.SelectedNodes.Count <= 1)
```

Comment:
Could this block be unified with its else block? At a glance without actually stepping through code, it looks like the else branch covers the general "N objects" case, including the "1 object" case of the if block. Can you check if a unified variant works as expected?